tests/MongoDB.Bson.Tests/IO/JsonWriterTests.cs

```diff
@@ -511,6 +511,24 @@ public void TestJavaScriptWithScope()
 #pragma warning restore 618, 1062
         }
 
+        [Fact]
+        public void TestGuidWhenStandardGuidRepresentationShouldBeTakenFromStandardSubType()
+        {
+            var guid = new Guid("00112233445566778899aabbccddeeff");
+            var standardGuidRepresentation = GuidRepresentation.Standard;
+            var standardSubType = GuidConverter.GetSubType(standardGuidRepresentation);
+            var guidBytes = GuidConverter.ToBytes(guid, standardGuidRepresentation);
+
+            var binary = new BsonBinaryData(guidBytes, standardSubType); // GuidRepresentation is Unspecified
+            var result = binary.ToJson(writerSettings: new JsonWriterSettings() // no exception
```

Comment:
I didn't understand what "no exception" was supposed to mean. Just remove it?

Reply:
yep, in the first version, I just asserted that there was no error, now it's not necessary comment